deploy/scripts/src/txma/test.ts

```diff
@@ -117,8 +150,4 @@ export function sendEvent (): void {
 export function sendEventDebug (): void {
```

Comment:
This function can be removed as it just duplicates the `sendEvent()` function

Reply:
Done